actions/run/src/run.ts

```diff
@@ -3,7 +3,8 @@ import { spawn } from 'child_process'
 import * as path from 'path'
 
 export async function run(): Promise<void> {
-  const artifact = core.getBooleanInput('artifact')
+  const artifactSaveLocal = core.getBooleanInput('artifact_save_local')
```

Comment:
I want to avoid this, if possible. These actions are used outside of CI in other places, so making backward incompatible changes will break them all. 

If I understand correctly, you want to explicitly set the artifact name instead of just dropping the artifacts into a folder. While this makes sense, I think it's unnecessary, because the artifact name is already set in the Earthfile. So, assuming you know the artifact name (which we do), you can use the action in the original form and just add an additional command afterwards:

```
mv ./artifacts/artifact_name /path/to/new/location
```

This would avoid making a breaking change.

Reply:
Yes, I think calling the artifact name directly would be much cleaner, but will revert and move the file instead.